meson.build

```diff
@@ -42,7 +42,7 @@ m_dep = meson.get_compiler('c').find_library('m', required : false)
 
 gresource_calendar = gnome.compile_resources(
     'gresource_calendar',
```

Comment:
Why is `gresource_calendar` used as name at this point?
Other than that I find this term only in calendar/src/meson.build (line 65).

Reply:
According to [the documentation](https://mesonbuild.com/Gnome-module.html), this argument represents the name of the resource which is used for a header file name generated in the `build` directory. In my understanding, it's not decided and any name is fine.